demo/demo-cse-v1/consumer/src/main/resources/application.yml

```diff
@@ -75,6 +75,54 @@ servicecomb:
           - weight: 80
             tags:
               version: 0.0.2
+      - precedence: 4
+        forceEnabled: true
+        match:
+          headers:
+            canary:
+              exact: force100
+        route:
+          - weight: 100
+            tags:
+              version: 0.0.3
+      - precedence: 5
+        forceEnabled: true
```

Comment:
Find a better name, e.g. `emptyProtection` and default to true. if `emptyProtection` is false, return empty list.

Reply:
fixed